test/unit/autocomplete.ts

```diff
@@ -215,6 +215,19 @@ suite('Autocomplete', ({ expect, spy }) => {
       expect(set).to.be.calledWith({ suggestions, navigations, categoryValues, selected: -1 });
       expect(setActivation).to.be.calledWith(targets, selected, false);
     });
+
+    it('should inactivate sayt when there are no suggestions', () => {
+      const suggestions = [];
+      const navigations = [];
+      const categoryValues = [];
+      const emit = autocomplete.flux.emit = spy();
```

Comment:
I would change this to 
```
const emit = spy();
autocomplete.flux = { emit };
```

this way there's no chance that we're modifying a value that is re-used across tests (`autocomplete.flux`)
We are also adding the requirement that `flux` be set somewhere in the tests, which would break our tests were it to ever change